source/pulsar.rst

```diff
@@ -78,7 +81,9 @@ Pulsar configuration
        - { 'src': 'patches/params_submission.patch', 'basedir': '/opt/pulsar/venv/lib/python2.7/site-packages', 'state': 'present', 'backup': 'yes' }
        - { 'src': 'patches/hostname.patch', 'basedir': '/opt/pulsar/venv/lib/python2.7/site-packages', 'state': 'present', 'backup': 'yes' }
 
-   where the ``<rabbit_mq_usegalaxy.eu_url>`` is the RabbitMQ url, retrieved from useGalaxy.eu.
+## is the UUID patch still needed?
```

Comment:
ping?

Reply:
Yes, John's UUID commit is available on GX20.01 and Pulsar0.14.dev1, which are both one step above what we are using now. Furthermore, it works with that GX version only. Instead, mine can be used also with previous versions.